.github/workflows/test.yml

```diff
@@ -496,6 +496,65 @@ jobs:
           TZ: ${{ vars.TIMEZONE }}
         run: npx cypress run --config video=false,screenshotOnRunFailure=false --project ./gef --record false --spec "./gef/cypress/e2e/${{ matrix.spec }}"
 
+
+  # 5.1 E2E - GEF backwards compatibility
+  e2e-tests-gef-feature-flag-backwards-compatibility:
+    name: E2E GEF FF COMPATIBILITY👷
+    needs: setup
+    environment:
+      name: ${{ needs.setup.outputs.environment }}
+    env:
+      GEF_DEAL_VERSION: 0
+    runs-on: ubuntu-latest
+
+    strategy:
+      # Do not cancel in-progress jobs upon failure
+      fail-fast: false
+
+    concurrency:
+      group: e2e-tests-gef-ff-compatibility-${{ github.workflow }}-${{ github.workflow_ref }}
+      cancel-in-progress: true
+
+    steps:
+      - name: Repository
+        uses: actions/checkout@v4
+
+      - name: Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.node }}
+
+      - name: Timezone
+        uses: szenius/set-timezone@v1.2
+        with:
+          timezoneLinux: ${{ needs.setup.outputs.timezone }}
+
+      - name: Dependencies
+        working-directory: ./
+        run: npm ci
+
+      - name: Docker
+        run: docker compose -f docker-compose.gha.yml up --build -d
+
+      - name: Initialise
+        working-directory: ./
+        run: |
+          sleep 100s
+          npm run load
+
+      - name: SQL
+        working-directory: ./libs/common
+        env:
+          SQL_DB_HOST: ${{ secrets.SQL_DB_HOST_LOCAL }}
+        run: npm run db:migrate
+
+      - name: Execute
+        working-directory: ./e2e-tests
+        env:
+          SQL_DB_HOST: ${{ secrets.SQL_DB_HOST_LOCAL }}
+          TZ: ${{ vars.TIMEZONE }}
+        run: npx cypress run --config video=false,screenshotOnRunFailure=false --project ./gef --record false --spec "./gef/cypress/e2e/facility/{about-facility,facilities-change-from-details-journey,provided-facility}.spec.js"
```

Comment:
This definitely isn't the most scalable but I want all of the tests to execute under one job

Reply:
Just feels a bit silly to build docker separately for each of these tests, rather than run them all under one job. I think it scales poorly either way:
- We use a matrix & have a separate job for each file, we could end up with 10+ jobs which would be inefficient computing-wise & potentially annoying when I have to scroll through 10 more tests on github
- We use the current set up, its a pain to work out what tests are currently being run & update them